index.html

```diff
@@ -0,0 +1,248 @@
+                <div id="3" class="column3 column"></div>
+            </div>
+            <div class="space">
+                <div id="4" class="column4 column"></div>
+            </div>
+            <div class="space">
+                <div id="5" class="column5 column"></div>
+            </div>
+            <div class="space">
+                <div id="6" class="column6 column"></div>
+            </div>
+
+        </div>
+
+        <div id="1" class="row1">
+
+            <div class="space">
+                <div id="0" class="column0 column"></div>
+            </div>
+            <div class="space">
+                <div id="1" class="column1 column"></div>
+            </div>
+            <div class="space">
+                <div id="2" class="column2 column"></div>
+            </div>
+            <div class="space">
+                <div id="3" class="column3 column"></div>
+            </div>
+            <div class="space">
+                <div id="4" class="column4 column"></div>
+            </div>
+            <div class="space">
+                <div id="5" class="column5 column"></div>
+            </div>
+            <div class="space">
+                <div id="6" class="column6 column"></div>
+            </div>
+    
+        </div>
+
+        <div id="2" class="row2">
+
+            <div class="space">
+                <div id="0" class="column0 column"></div>
+            </div>
+            <div class="space">
+                <div id="1" class="column1 column"></div>
+            </div>
+            <div class="space">
+                <div id="2" class="column2 column"></div>
+            </div>
+            <div class="space">
+                <div id="3" class="column3 column"></div>
+            </div>
+            <div class="space">
+                <div id="4" class="column4 column"></div>
+            </div>
+            <div class="space">
+                <div id="5" class="column5 column"></div>
+            </div>
+            <div class="space">
+                <div id="6" class="column6 column"></div>
+            </div>
+
+        </div>
+
+        <div id="3" class="row3">
+
+            <div class="space">
+                <div id="0" class="column0 column"></div>
+            </div>
+            <div class="space">
+                <div id="1" class="column1 column"></div>
+            </div>
+            <div class="space">
+                <div id="2" class="column2 column"></div>
+            </div>
+            <div class="space">
+                <div id="3" class="column3 column"></div>
+            </div>
+            <div class="space">
+                <div id="4" class="column4 column"></div>
+            </div>
+            <div class="space">
+                <div id="5" class="column5 column"></div>
+            </div>
+            <div class="space">
+                <div id="6" class="column6 column"></div>
+            </div>
+
+        </div>
+
+        <div id="4" class="row4">
```

Comment:
technically, if used for actual selectors, this ID 4 would conflict with the ID 4 inside.